packages/graz/src/actions/account.ts

```diff
@@ -147,6 +155,14 @@ export const connect = async (args?: ConnectArgs): Promise<ConnectResult> => {
 export const disconnect = (args?: { chainId?: ChainId }) => {
   typeof window !== "undefined" && window.sessionStorage.removeItem(RECONNECT_SESSION_KEY);
   const chainId = typeof args?.chainId === "string" ? [args.chainId] : args?.chainId;
+
+  const walletConnectInstance = getWallet(WalletType.WALLETCONNECT);
+  const { disable: walletConnectDisable } = walletConnectInstance;
+
+  if (walletConnectDisable) {
+    void walletConnectDisable();
+  }
+
```

Comment:
you need to wrap this if the wallet type they are trying to connect with is wallet connect